lua/dap-view/util/exprs.lua

```diff
@@ -0,0 +1,25 @@
+local M = {}
+
+M.get_current_expr = function()
+    if vim.fn.mode() == 'v' then
+        local start = vim.fn.getpos('v')
+        local finish = vim.fn.getpos('.')
+        local start_line, start_col = start[2], start[3]
+        local finish_line, finish_col = finish[2], finish[3]
+
+        if start_line > finish_line or (start_line == finish_line and start_col > finish_col) then
+            start_line, start_col, finish_line, finish_col = finish_line, finish_col, start_line, start_col
+        end
+
+        local lines = vim.fn.getline(start_line, finish_line)
+        if #lines == 0 then
+            return ""
+        end
```

Comment:
```suggestion
        if #lines == 0 then
            return ""
        end

        if type(lines) == "string" then
            return lines
        end

```

LuaLS is giving a warning because `getline` may return a single string

Reply:
Will change